### PR TITLE
Improve mongo db connection handling

### PR DESCRIPTION
**Pull request by Tumii01**
Added Necessary Database Modules to meet the issues check List.

### CODE REVIEW

File: database/database.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+const client = new MongoClient(uri, {
+  serverApi: {
+    version: ServerApiVersion.v1,
+    strict: true,
+    deprecationErrors: true,
+  }});
+
+  async function connectToDatabase() {
```
Comment (Renzo-codespace) [inline database/database.js:21]: Returning client.db this way is a bit of an unusual practice.
But in theory this shouldn't cause a spike in the number of connections (I say this under correction).

Comment (Renzo-codespace) [reply database/database.js:21]: No further action required now on this point.
You may resolve this conversation after you've made a mental note of my comment.

**Review by Renzo-codespace — APPROVED** [review]
Good work.
Even though returning client.db is unusual, it does add a certain layer of protection.
Your implementation is consistent across the code base. That's great.
Please help the rest of your team understand these changes so they may use your implementation in their own contributions.

**Review by NelsonMALGAS — APPROVED** [review]
I can see that client is only used in one file like it was specified

**Pull request merged**